api/src/main/resources/application-context.xml

```diff
@@ -161,6 +158,14 @@
         <property name="jdbcTemplate" ref="jdbcTemplate"/>
     </bean>
 
+    <bean id="mailsMap" class="org.springframework.beans.factory.config.PropertiesFactoryBean" >
+        <property name="locations">
+            <list>
+                <value>file:///etc/perun-spRegistration/mailsMap.properties</value>
```

Comment:
Also here change the name of the file to something more descriptive.

Reply:
Done